docs/community/contribution guidelines.rst

```diff
@@ -128,6 +128,118 @@ Create a pull request (from now on we will shorten it often to just *PR*) by vis
+|   {                                   |       {                                  |
+|   };                                  |       };                                 |
+|                                       |                                          |
+|   }                                   |   }                                      |
+|                                       |                                          |
++---------------------------------------+------------------------------------------+
+
+Operator spacings
+-----------------
+
++---------------------------------------+------------------------------------------+
+| Correct                               | Incorrect                                |
++=======================================+==========================================+
+|                                       |                                          |
+|.. code-block:: c++                    |.. code-block:: c++                       |
+|   :emphasize-lines: 6                 |   :linenos:                              |
+|   :linenos:                           |                                          |
+|                                       |                                          |
+|   // Unary                            |   // Unary                               |
+|   i++;                                |   i ++;                                  |
+|   if (!b) {}                          |   if (! b) {}                            |
+|                                       |                                          |
+|   // Assignment, binary, ternary      |   // Assignment, binary, ternary         |
+|   y = m*x + b;                        |   y = m*x+b;                             |
+|   c = a | b;                          |   c=a|b;                                 |
+|   return condition ? true : false;    |   return condition?true:false;           |
+|                                       |                                          |
++---------------------------------------+------------------------------------------+
+| | To emphasize order no spacing is OK |                                          |
+|   sometimes                           |                                          |
+| | (line no 6)                         |                                          |
++---------------------------------------+------------------------------------------+
+
+Braces & more spaces
+--------------------
+
+.. code-block:: c++
+
+   /* All C++ reserved keywords where braces are inevitable
+    * Are placed on the same line
+    *
+    * Also make sure to leave one space before function brackets
+    */
+
+   if (condition) {
+       ...
+   } else {
+       ...
+   }
+
+   switch (value) {
+       case 1: ...
+       case n: ...
+   }
+
+   enum MyEnum {
+       ...
+   };
+   
+   // Same apply for C++ loops too
+
+   while (condition) { ... }
```

Comment:
avoid single line :)

Reply:
Wanted to keep the section topics minimal so I just lumped the below discussion in this portion too ;P

For consistency's sake, a new separate section would be much better. Updating the doc...

![Screenshot_2023-05-23_16-48-47](https://github.com/synfig/synfig-docs-dev/assets/24214509/cffad136-0742-46cf-932b-2f5f88793c5f)